cmd/flags.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"errors"
+	"strings"
+)
+
+// ErrorEmpty returned when value is empty.
+var ErrorEmpty = errors.New("empty value not allowed")
+
+// StringSlice is a flag that allows to set multiple types of string
+type StringSlice []string
+
+func (s *StringSlice) String() string {
+	return "string slice"
+}
+
+// Set unmarshals enode into discv5.Node.
```

Comment:
I don't see `discv5.Node` reference anywhere in this file. Isn't it more general?

Reply:
it was copy paste